editor/reducer.js

```diff
@@ -426,22 +433,16 @@ export function showInsertionPoint( state = false, action ) {
 }
 
 /**
- * Reducer returning current editor mode, either "visual" or "text".
+ * Reducer returning the user preferences:
+ *   - current editor mode, either "visual" or "text".
```

Comment:
Minor: Should we prefix these list items with the names of the keys they correspond to, e.g.

```
 *  - mode: current edidtor mode, either "visual" or "text".
```

Alternatively, it's also possible to document object shapes with JSDoc:

http://usejsdoc.org/tags-type.html#jsdoc-types